.github/workflows/lint.yml

```diff
@@ -0,0 +1,36 @@
+name: Lint
+
+on:
+  push:
+    paths:
+      - '*.py'
+      - '!**.md'   
+  pull_request:
+    branches:
+      - master
+    paths:
+      - '*.py'  
+      - '!**.md'  
+  release:
+    types:
+      - released    
+
+jobs:
+  flake8_py3:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Setup Python
+        uses: actions/setup-python@v1
+        with:
+          python-version: 3.7.4
+          architecture: x64
+      - name: Checkout PyTorch
```

Comment:
Are we using PyTorch here?

Reply:
Using template from https://github.com/marketplace/actions/flake8-action